specificatie/genereervariant/openapi.json

```diff
@@ -314,6 +314,72 @@
               "type": "string"
             }
           },
+          {
+            "name": "postcode",
+            "in": "query",
+            "description": "Postcode behorende bij een nummeraanduiding.",
```

Comment:
kunnen we dit veranderen in: de postcode van het adres

Reply:
wat voegt deze description toe? kan je ook postcodes hebben van iets anders dan een adres?